bin/ci/script.sh

```diff
@@ -11,9 +11,7 @@
 
 cd "$(dirname "$0")" || exit; source _includes.sh
 
-if [[ "$ORCA_JOB" ]]; then
-  eval "orca ci:run $ORCA_JOB script $ORCA_SUT_NAME"
-fi
+
```

Comment:
Just double-checking: was this a deliberate change to move this? Doesn't seem like it would make much difference in the end.